Classes/Hooks/ContentPostProcessorHook.php

```diff
@@ -123,7 +123,12 @@ protected function getConfiguration(array $rawConfiguration) {
     foreach ($configuration as $option => $value) {
 
       // Perform stdWrap processing on all options
-      if (StringUtility::isLastPartOfString($option, '.') && isset($value['stdWrap.'])) {
+      if (version_compare(TYPO3_version, '7.0', '>=')) {
```

Comment:
The benefit is not performing exactly the same check with exactly the same result over and over again. :-)